### DISCUSSION TITLE
email language texts: line breaks and html br p

### DISCUSSION THREAD

**Discussion by torvista** [zencart/zencart#5818] [Ideas]
Seeing that the text in a password forgotten email was all one block, I looked at the language define: the line breaks are \n\n.
Looking at zen_mail I don't see where there is any substitution to br for html emails.

There have discussions regarding formatting.
https://github.com/zencart/zencart/issues/2742
https://github.com/zencart/zencart/issues/3757

Would it not be easier to allow html in the define and remove/replace html for text-only emails?

**Comment by scottcwilson** [comment]
I'm hoping the work being planned in #5499 will resolve things like this.

**Comment by lat9** [comment]
Can this discussion be combined with its predecessor (https://github.com/zencart/zencart/discussions/5814)?
